docs/kb/KHV050.md

```diff
@@ -0,0 +1,22 @@
+---
+id: KHV050
+title: Read access to Pod service account token
+categories: [Access Risk]
```

Comment:
At some point we should probably have a reference about what we mean by these different categories. Can certainly be in a separate PR, I'm just mentioning it while I see it